src/desktop/index.js

```diff
@@ -21,6 +21,7 @@ app.use(
 app.use(require("./apps/home"))
 app.use(require("./apps/editorial_features"))
 app.use(require("./apps/apply"))
+app.use(require("./apps/auction_reaction/server").app)
```

Comment:
Do we prefer `server` over `index` now? Or is requiring a server file for different use-cases?